packages/cli/internal/pkg/stringutils/str_utils.go

```diff
@@ -1,4 +1,4 @@
-package slices
+package stringutils
```

Comment:
I would be more specific and call it something like `unicode`. It would better document your intent.